.gitignore

```
@@ -63,21 +63,31 @@ cpu.out
 /indexers
 /log
 /public/img/avatar
-/integrations/gitea-integration-mysql
-/integrations/gitea-integration-mysql8
-/integrations/gitea-integration-pgsql
-/integrations/gitea-integration-sqlite
-/integrations/gitea-integration-mssql
-/integrations/indexers-mysql
-/integrations/indexers-mysql8
-/integrations/indexers-pgsql
-/integrations/indexers-sqlite
-/integrations/indexers-mssql
-/integrations/sqlite.ini
-/integrations/mysql.ini
-/integrations/mysql8.ini
-/integrations/pgsql.ini
-/integrations/mssql.ini
+/tests/integration/gitea-integration-mysql
+/tests/integration/gitea-integration-mysql8
+/tests/integration/gitea-integration-pgsql
+/tests/integration/gitea-integration-sqlite
+/tests/integration/gitea-integration-mssql
+/tests/integration/indexers-mysql
+/tests/integration/indexers-mysql8
+/tests/integration/indexers-pgsql
+/tests/integration/indexers-sqlite
+/tests/integration/indexers-mssql
+/tests/e2e/gitea-e2e-mysql
+/tests/e2e/gitea-e2e-mysql8
+/tests/e2e/gitea-e2e-pgsql
+/tests/e2e/gitea-e2e-sqlite
+/tests/e2e/gitea-e2e-mssql
+/tests/e2e/indexers-mysql
+/tests/e2e/indexers-mysql8
+/tests/e2e/indexers-pgsql
+/tests/e2e/indexers-sqlite
+/tests/e2e/indexers-mssql
+/tests/sqlite.ini
+/tests/mysql.ini
+/tests/mysql8.ini
+/tests/pgsql.ini
+/tests/mssql.ini
```

List is becoming quite long
```suggestion
/tests/integration/gitea-integration-*
/tests/integration/indexers-*
/tests/e2e/gitea-e2e-*
/tests/e2e/indexers-*
/tests/*.ini
```